golang-builder/Dockerfile.template

```diff
@@ -8,23 +8,48 @@ ENV GOCACHE="${GOPATH}/build-cache"
 RUN mkdir GOCACHE
 
 RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.9/main' >> /etc/apk/repositories; \
-    echo 'http://dl-cdn.alpinelinux.org/alpine/v3.9/community' >> /etc/apk/repositories; \
-    apk --update add --no-cache mongodb yaml-cpp=0.6.2-r2 ca-certificates bash httpry make git curl python py-pip docker \
-    rhash openssh unzip libc6-compat openjdk8-jre gcc musl-dev python-dev libffi-dev openssl-dev libc-dev; \
-    pip install docker-compose; \
-    pip install awscli;
+	echo 'http://dl-cdn.alpinelinux.org/alpine/v3.9/community' >> /etc/apk/repositories; \
```

Comment:
can we have a consistent character for string?, this make inconsistencies. you can choose to use ' or "

Reply:
using double quotes now